apps/api/src/modules/payments/index.ts

```diff
@@ -97,6 +102,20 @@ export async function paymentRoutes(app: FastifyInstance) {
       },
       getPaymentById
     )
+    .get(
+      '/:paymentId/admin',
```

Comment:
Is there a reason not to nest all admin-only api routes under a top level admin/ namespace? Tacking admin on the end doesn't feel in keeping with rest conventions.

Reply:
As far as the back-end is concerned they aren't admin routes, though we could still organize that way. I was unsure how to deal with this route, as it has additional information returned, but otherwise is just a normal endpoint for getting payment details. I added "admin" but that doesn't feel appropriate necessarily. Any thoughts on how we should organize?

Reply:
Ah I gotcha. Good call. Hrm. This is probably fine then. The only other thing I can think of is something like "with-x" if all it did was return some additional relation.

Reply:
Aw yeah, good point. That makes more sense - I can convert to that structure.